manifests/03-rbac-role-cluster.yaml

```diff
@@ -68,3 +68,16 @@ rules:
   - create
   - update
   - delete
+---
```

Comment:
So I've just realised that it would be better to put this into `03-rbac-role-cluster-extensions.yaml` file, since that contains extensions.

Reply:
It was not added to console extensions manifest as it would get applied to every user. And that would make CRDs listable directly by every user. We are trying to avoid this. Hence `03-rbac-role-cluster.yaml` is better place for this. This was discussed with @spadgett